processing/src/test/java/org/apache/druid/java/util/common/guava/ParallelMergeCombiningSequenceTest.java

```diff
@@ -285,6 +285,27 @@ public void testEmptiesAndNonEmpty() throws Exception
     input.add(nonBlockingSequence(5));
     assertResult(input);
   }
+  @Test
```

Comment:
am a bit surprised this didnt get flagged by the style stuff for not having a space between the methods